selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/CheckRestoringSplitEditorTest.java

```diff
@@ -148,12 +152,4 @@ private void setPositionsForSplittedEditor() {
     editor.selectTabByName(pomFileTab);
     editor.goToPosition(cursorPositionForPomFile.first, cursorPositionForPomFile.second);
   }
-
-  private String getPreferences() throws Exception {
```

Comment:
It may be reasonable to move getPreferences() method to the **TestUserPreferencesServiceClient** class.